Highlight the differences between scripts and modules (#1567)
nushell/nushell.github.io: book/scripts.md

## Subcommands

> A script's `main` command can have multiple [subcommands](custom_commands.html#subcommands) like `run`, `build`, etc. which allows executing a specific sub-function from the commandline.

Reviewer: ```suggestion
A script can have multiple [subcommands](custom_commands.html#subcommands) like `run` or `build`.
```

To the user of the script, it's a subcommand of the script, not of `main`.  I think the original wording is probably better here, but still like your other change here.

    print "building"
}

def main [] {
    print "hello from myscript!"
}
```

> 

Reviewer: ```suggestion
You can then execute the script's subcommands when calling it:
```
